merlin/common/tasks.py

```diff
@@ -96,8 +96,9 @@ def merlin_step(self, *args: Any, **kwargs: Any) -> Optional[ReturnCode]:  # noq
 
     if step:
         self.max_retries = step.max_retries
-        step_name: str = step.name()
-        step_dir: str = step.get_workspace()
+        step_name = step.name()
+        step_dir = step.get_workspace()
+        LOG.debug(self.request.id)
```

Comment:
are these two debug messages giving the same info?

Reply:
One is displaying the id and the other is displaying the step_name and step_dir but since the dir is the same as the id I think I can remove it.

Reply:
yeah. since the step_id is useful to know, maybe combine and say
LOG.debug(f"Running Step {step_name} -- Step ID: {self.request.id}")

Reply:
And as I look below at the various other LOG files, I think we should just replace all the directory calls with {self.request.id}
Like LOG.info("fSkipping step {self.request.id}")
I think having the step_name and the step_id in there at the same time could be confusing